_config.yml

```diff
@@ -177,11 +180,8 @@ navigation:
       - children:
           - text: Who we are
             children:
-              - text: General contacts and listservs
-                url: general-contacts-and-listservs/
```

Comment:
Can we put in a redirect so external links don't break? Let me know if you need help with that.

Reply:
I think that's a very good idea.

Reply:
@afeld now that the redirect is in, can you take another look please?